### DISCUSSION TITLE
return null value as null instead of empty string

### DISCUSSION THREAD

**Discussion by hebrd** [graphql-java/graphql-java#3681] [Q&A]
I have a type
```
type Dict {
...
updateUser: Int
...
}
```

```java
public class DictEntity {
@Schema(

        name = "updateUser",
        nullable = true
    )
    private Long updateUser;

}
```

When the updateUser is null, the returned json is blank string "", how can I return null instead of ""?
```
{
          "code": " I am code",
          "createTime": "2024-08-07T23:08:07",
          "updateUser": "",
          "orgId": 1
        }
```

**Comment by bbakerman** [comment]
You seem to be using some framework - not sure what annoation `@Schema` is from?  Can you expand more on that

Also your schema type is `updateUser: Int` yet your Java type is a Long and the  Json value is String -  that strikes me as weird but since I have no idea what framework the above is using its hard to know if thats expected or not
